packages/react/src/themes/teams/components/SplitButton/splitButtonVariables.ts

```diff
@@ -1,29 +1,30 @@
 import { SiteVariablesPrepared } from '../../../types'
-
 import { pxToRem } from '../../../../lib'
 
 export interface SplitButtonVariables {
   borderRadius: string
-  backgroundColorFocus: string
-  boxShadow: string
+  borderColorPrimary: string
   borderColor: string
-  colorFocus: string
-  primaryBackgroundColorFocus: string
-  primaryColorFocus: string
+  borderColorDisabled: string
+  borderWidthLeftOnly: string
+  smallDimension: string
```

Comment:
I don't see a dimension variable for the default button size. If we need one for `small` size should there be one for the default size?

Reply:
It's inherited by Button's size, so I think we just keep it as-is unless we remove the dependency on Button...